### PR TITLE
Testnet and Dev Tesnet now deploy from one Workflow

### PR DESCRIPTION
**Pull request by otherview**
### Why is this change needed?

- https://github.com/obscuronet/obscuro-internal/issues/697

### What changes were made as part of this PR:

- Provide a high level list of the changes made

### Definition of done

- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated 
- [ ] End-to-end tests run if required (see below)

### End-to-end tests

Running the end-to-end tests in the [obscuro-test repo](https://github.com/obscuronet/obscuro-test) is currently at the 
discretion of the developer prior to merging a PR. The intention is not to slow down development by having the longer 
running end-to-end tests as a PR gate run on each branch commit etc. To manually trigger a run pre-merge;

- Go to [run_local_tests](https://github.com/obscuronet/obscuro-test/actions/workflows/run_local_tests.yml)
- Click the "Run workflow" button
- Use the main branch of obscuro-test 
- Enter the name of the PR branch for go-obscuro
- Run the workflow

The run takes approximately 20 minutes and any failure will be reported in the workflow output. Should the tests fail 
the docker container output for the local testnet and all test artifacts will be added to the run and can be downloaded
for debugging and analysis, with a retention period of 2 days. 

Note that every PR merge to main will also trigger a run of post-merge tests, so even if you do not manually trigger 
pre-merge, tests will be run post-merge. The intention being to catch any issues fast on main to allow for quick 
resolution. The post-merge test output can be seen at 
[run_merge_tests](https://github.com/obscuronet/obscuro-test/actions/workflows/run_merge_tests.yml) and will show both 
the PR number and author in the list of runs.

### CODE REVIEW

File: .github/workflows/manual-deploy-testnet-l2.yml [hunk 1]
```diff
@@ -30,47 +45,80 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
-      - name: 'Build and push obscuro node images'
+      - name: 'Sets env vars for tesnet'
```
Comment (moraygrieve) [inline .github/workflows/manual-deploy-testnet-l2.yml:48]: testnet not tesnet

File: .github/workflows/manual-deploy-testnet-l2.yml [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
+        type: choice
+        options:
+          - 'dev-testnet'
+          - 'testnet'
       L1HOST:
-        description: 'geth network fqdn address'
+        description: 'L1 FQDN Address (defaults to Dev Testnet L1)'
         required: true
-        default: 'testnet-gethnetwork.uksouth.azurecontainer.io'
+        default: 'dev-testnet-gethnetwork.uksouth.azurecontainer.io'
```
Comment (moraygrieve) [inline .github/workflows/manual-deploy-testnet-l2.yml:22]: `dev-testnet-gethnetwork.uksouth.azurecontainer.io`

that is specific to dev-testnet though isn't it?

File: .github/workflows/manual-deploy-testnet-l1.yml [hunk 3]
```diff
@@ -8,21 +8,44 @@
...
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Docker
+      - name: 'Set up Docker'
         uses: docker/setup-buildx-action@v1
 
+      - name: 'Sets env vars for tesnet'
```
[the remark this answers is not in the document]
Comment (otherview) [reply .github/workflows/manual-deploy-testnet-l1.yml:37]:
```suggestion
      - name: 'Sets env vars for testnet'
```

File: .github/workflows/manual-deploy-testnet-l2.yml (hunk 1 above)
Comment (otherview) [reply .github/workflows/manual-deploy-testnet-l2.yml:48]:
```suggestion
      - name: 'Sets env vars for testnet'
```

File: .github/workflows/manual-deploy-testnet-l2.yml (hunk 2 above)
Comment (moraygrieve) [reply .github/workflows/manual-deploy-testnet-l2.yml:23]: so what would it be for testnet?

Comment (otherview) [reply .github/workflows/manual-deploy-testnet-l2.yml]: `testnet-gethnetwork.uksouth.azurecontainer.io`

But we prolly don't need the ability to specifying the L1 Host. ( We haven't really used so far).
So I'm removing this input.

**Review by moraygrieve — APPROVED** [review]

**Pull request merged**